templates/tfengine/recipes/cicd.hcl

```diff
@@ -42,7 +42,15 @@ schema = {
       }
     }
     cloud_source_repository = {
-      description          = "Config for Google Cloud Source Repository Cloud Build triggers."
+      description          = <<EOF
+        Config for Google Cloud Source Repository Cloud Build triggers.
+
+        IMPORTANT: Cloud Source Repositories does not support code review or
+        presubmit runs. If you set both plan and apply to run at the same time,
+        they will conflict when trying to acquire the lock and may error out.
```

Comment:
I'm not sure if it's related to the lock as tf-plan has --lock set to false. Maybe just say "they will conflict and may error out."